DistributedLock.Tests/Infrastructure/TestingLockProvider.cs

```diff
@@ -19,6 +19,8 @@ public abstract class TestingLockProvider<TStrategy> : ITestingNameProvider, IDi
         public virtual string GetCrossProcessLockType() => this.CreateLock(string.Empty).GetType().Name;
         public virtual void Dispose() => this.Strategy.Dispose();
 
+        public string? GetLockPrefix() => Strategy.GetLockPrefix();
```

Comment:
Why is this needed?

Reply:
The need to run RunLockTaker / DistributedLockTaker with global lock name (prefix + local name) in order to pass cross process tests.